README.md

```diff
@@ -23,7 +24,7 @@ Example of generating a client library for Language API:
 git clone https://github.com/googleapis/googleapis
 cd googleapis
 bazel build //google/cloud/language/v1:google-cloud-language-v1-java
-bazel build //google/cloud/language/v1:gapi-cloud-language-v1-go
+bazel build //google/cloud/language/v1:language_go_gapic
```

Comment:
Actually it is right the way it was - the target it was previously referencing is the `go_gapic_assmebly_pkg` which assembles _all_ of the generated code (gapic + protobuf/grpc). Please revert this.